kolibri/plugins/learn/assets/src/actions.js

```diff
@@ -38,6 +56,12 @@ const nodeAssignment = (nodes, dispatch) => {
  * @param {String} id - The id of the model to fetch the children of.
  */
 const fetchNodes = ({ dispatch }, id) => {
+  if (id === undefined) {
```

Comment:
Also wondering if we should put the bootstrapped topics into the ContentNode cache somehow too.